x-pack/plugins/transform/server/routes/api/transforms.ts

```diff
@@ -316,6 +322,46 @@ export function registerTransformsRoutes(routeDependencies: RouteDependencies) {
     )
   );
 
+  /**
+   * @apiGroup Transforms
+   *
+   * @api {post} /api/transform/reset_transforms Post reset transforms
+   * @apiName ResetTransforms
+   * @apiDescription resets transforms
+   *
+   * @apiSchema (body) resetTransformsRequestSchema
+   */
+  router.post<undefined, undefined, ResetTransformsRequestSchema>(
```

Comment:
Can you add an API test for this new endpoint?

Reply:
Added API tests in ab14042dd36.